scylla/src/statement/batch.rs

```diff
@@ -178,6 +178,12 @@ impl From<&str> for BatchStatement {
     }
 }
 
+impl From<String> for BatchStatement {
+    fn from(s: String) -> Self {
+        BatchStatement::Query(Query::from(s))
+    }
+}
+
```

Comment:
Was this change required to implement the test (it's in the same commit)?

Reply:
Good catch. I included this by accident, after I noticed such implementation is missing.

Reply:
I'm move this implementation out of this PR.